requirements/base.txt

```diff
@@ -27,4 +27,4 @@ sortedcontainers==2.3.0   # via -r requirements/base.in
 unidecode==0.4.21         # via awesome-slugify
 
 # The following packages are considered to be unsafe in a requirements file:
-# pip
+pip==20.3.1               # via -r requirements/base.in, pip-tools
```

Comment:
This is again a remnant of our old issue right?

Reply:
Yes right @hannesdiedrich